src/custom-elements/src/components/date/date.tsx

```diff
@@ -74,26 +74,29 @@ export class Date {
   @Watch('excludedDates')
   updateExcludedDates() {
     this.excludedDatesArray = this.excludedDates
-    ? this.excludedDates.split(',').map(date => date.trim())
-    : [];
+      ? this.excludedDates.split(',').map(date => date.trim())
+      : [];
   }
 
   @Watch('excludedWeekdays')
   updateExcludedWeekdays() {
     this.excludedWeekdaysArray = this.excludedWeekdays
-    ? this.excludedWeekdays.split(',').map(weekDay => parseInt(weekDay))
-    : [];
+      ? this.excludedWeekdays.split(',').map(weekDay => parseInt(weekDay))
+      : [];
   }
 
   @Watch('value')
   dateChanged(newValue: string, oldValue: string) {
     if (newValue !== oldValue && (newValue || oldValue)) {
       if (newValue) {
         if (this.multiple) {
-          this._vm.dates = this.value ?
-            this.value.split(',').map(valueDay => this.fromString(valueDay)) : [];
+          this._vm.dates =
+            this.value.split(',').map(valueDay => this.fromString(valueDay)) ||
+            [];
         } else {
-          this._vm.dates = [this.fromString(this.value)];
+          const date = dayjs(this.value).format(this.format);
```

Comment:
I just realised about this, sorry.. but since you are watching for this.value already.. you can use it as newValue within the method, and not use this.value --> this.value replaced by newValue